pyproject.toml

```
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "fence"
-version = "4.22.0"
+version = "4.22.1"
```

```suggestion
version = "4.26.0"
```
latest is 4.25.0, and this is a new feature!